### DISCUSSION TITLE
What's your git personality?

### DISCUSSION THREAD

**Discussion by rmw** [community/events#48] [Cloud]
If you caught my talk, [GitHub Desktop is for Everyone](https://githubuniverse.com/events/detail/virtual-schedule/d98d21b8-2c51-4dc6-b), you know that I talk about different git personalities and use [Rainbow Brite's Color Kids](https://rainbowbrite.fandom.com/wiki/The_Color_Kids) to illustrate this. And if you missed the talk, [check it out tomorrow](https://githubuniverse.com/events/detail/virtual-schedule/d98d21b8-2c51-4dc6-b) at 2:45pm CST.

Are you like ...
❤️ 🔥  Red Butler - a little fiery, a little spicy and someone who likes to experiment. you might have a like of branches and tend to mess with your commit history before sharing it.
🧡 🎨  LaLa Orange - creative, expressive, and very focused on collaborating with your amazing teammates. co-authoring FTW!
💙 💪 Blue Butler - particular about how you do things and focused on the best, most efficient workflow, often mixing and matching tools.
💜 👩‍💻 Shy Violet - nerdy and loves to philosophize. you're writing strong commit messages and like your workflow but are willing to try new things when they make sense.

What's your git personality?

**Comment by ossanna16** [comment]
Oh this is a fun one! I'm a mix of 🧡🎨 and 💙💪.

**Comment by queenofcorgis** [comment]
❤️ 🔥  - definitely ! Can't wait for your session @rmw

**Comment by jricciardi** [comment]
🧡🎨 for sure!

**Comment by samus-aran** [comment]
Definitely 💙 💪 🔥

**Comment by Bestra** [comment]
I think I'm 💙 💪 with touches of 🧡🎨

**Comment by alefred** [comment]
💜 that like experiment like ❤️... 😁🤘

**Comment by stevepopovich** [comment]
❤️ 🔥 you already know ❤️ 🔥 🌶️  great talk, caught most of it!

**Reply by rmw** [reply]
I'm SHOCKED. Just shocked.

J/K. ❤️

**Comment by mishmanners** [comment]
❤️ all the way!

**Comment by hrthealer** [comment]
I agree a fun one, 💙💪, 🧡 🎨, ❤️ 🔥; very new to this coding adventure.  Love it, a love of mine has always been computer language like C, pascal and Fortran (not a whole lot, just the basics).  I got into healthcare and would love to share some of my ideas and experiences as well as support, contribute and maintain (I am new 🤓)

**Comment by mxie** [comment]
Most definitely 💙💪 with hints of 🧡🎨! 😊

**Comment by Patecatl848** [comment]
I could honestly say I am ❤️ 🔥, 🧡 🎨 &  💙 💪

**Comment by K0I0S-Codes** [comment]
Saw the talk! Loved it and the illustrations and I am probably 💜 👩‍💻 & ❤️ 🔥
